app/models/group.rb

```diff
@@ -9,7 +12,9 @@ class Group < ActiveRecord::Base
   scope :students, -> { where(name: 'Students') }
   scope :coaches, -> { where(name: 'Coaches') }
 
-  validates :name, :chapter_id, presence: true
+  validates :name, presence: true
+
+  validate :valid_name?
```

Comment:
Of course, good point. Shows how rusty my rails knowledge is, I had completely forgotten abt inclusion. Thx for reviewing!